### ISSUE TITLE
Clean up add content interface

### ISSUE THREAD

**Issue by augustf** [concerto/concerto-simple-rss#43]
![screen shot 2015-08-21 at 2 39 10 pm](https://cloud.githubusercontent.com/assets/12512/9416414/7cb5ca48-4813-11e5-86e7-41124282bf92.png)

We should make some changes to make the interface more obvious and accommodating to a novice user. In particular "Blacklisted HTML Tags for Sanitization" and "Display Format" are unclear. It might be better to have checkboxes for "allow images" or "allow links" to regulate tag filtering.

**Comment by mfrederickson** [comment]
I don't agree with the checkbox approach because that is too restrictive as to what the consumer may want stripped out.

**Comment by augustf** [comment]
I'm not married to any one approach in particular. But I bring this ticket up to try to push us back towards a somewhat minimal interface. Perhaps we can consider hiding more advanced options in the same manner we hide the time controls for graphical content. Or just rejigger the fields in a more explanatory way that makes it clearer to a user who isn't familiar with advanced web applications what's going on.
